odometry_localization:
  ros__parameters:
    update_rate: 200.0
    integration_rate: 100.0
    wheel_base: 0.173
    wheel_radius: 0.0505

pid_path_generator:
  ros__parameters:
    waypoints_json: |
      [
        { "x": 0.6, "y": 0.0 },
        { "x": 0.6, "y": 0.6 },
        { "x": 0.0, "y": 0.6 },
        { "x": 0.0, "y": 0.0 }
      ]
    min_waypoint_distance: 0.1
    max_waypoint_distance: 2.0

pid_point_controller:
  ros__parameters:
    update_rate: 100.0
    Kp_V: 0.15
    Ki_V: 0.0
    Kd_V: 0.0
    Kp_Omega: 0.1
    Ki_Omega: 0.0
    Kd_Omega: 0.0
    goal_tolerance: 0.08
    heading_tolerance: 0.08
    min_linear_vel: 0.1
    max_linear_vel: 0.17
    min_angular_vel: -1.0
    max_angular_vel: 1.0
    auto_request_next: true
    use_constant_linear_speed: false